recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, CMake
+from conan.tools.files import get, collect_libs
+
+
+class OctoLoggerCPPConan(ConanFile):
+    name = "octo-logger-cpp"
+    version = "1.0.0"
```

Comment:
```suggestion
```
Don't add a specific version to the recipe, so it can be re-used for any future version.

Reply:
was for testing, forgot to remove, fixed, thanks!